### ISSUE TITLE
Helm chart to deploy 

### ISSUE THREAD

**Issue by rohitjha941** [pixie-io/pixie#431]
I am entirely new to Pixie. Please Pardon me if things mentioned below are already present. 

**Is your feature request related to a problem? Please describe.**
I am looking for a Helm Chart to deploy Pixie Cloud to Kubernetes.  My Infrastructure is wholly automated using terraform. I will prefer to continue using Terraform for it. Current Install Instructions are tough to automate. A helm chart that can provision pixie cloud on the existing cluster will be helpful. 

Another issue is `/scripts/create_cloud_secrets.sh` generates a self-signed certificate with a pixie domain name. The ability to take an external certificate or Integration with [cert-manager](https://github.com/cert-manager/cert-manager) will be helpful. 

**Describe the solution you'd like**
A helm chart will be helpful. 

**Describe alternatives you've considered**
I could use the Kustomise module to carry out steps 7, 8, and 9. And can recreate all secrets using Kubernetes and  TLS providers in terraform.

**Comment by laxmansoma1903** [comment]
@zasgar - we are trying to perform similar as above , can you please provide the timeline if possible on when we can have helm charts available..

**Comment by laxmansoma1903** [comment]
Please let us know if you able to achieve this,.....

**Comment by laxmansoma1903** [comment]
Hi Starostin - Did you able to make it work?

**Comment by s-starostin** [comment]
Hello, yes, but there were some specific defaults like nodeSelector and tolerations. Removed them now in values, also added small README in the chart dir and elasticsearch chart repo to the Chart.yaml.

**Comment by EStork09** [comment]
I think outside of the helm charts that people have already created. Is there any desire from the pixie-labs maintainers if we created a chart similar to how the operators has one. If we created would that be maintained similar to the kustomize, or should someone create a chart for pixie cloud outside of the pixie labs umbrella?

**Comment by miospruit** [comment]
i would say it would be very nice to get a helm chart of the pixie cloud from the developers of pixie.

**Comment by RickFoland** [comment]
I'd also be interested in a helm chart for Pixie Cloud that is maintained by Pixie. The one for the operator is nice, but it would be great to go fully self-hosted using helm. 

As the original post states, the current process for deploying Pixie Cloud is not easy to automate and I'd prefer not to rely on a third party chart. Having a Pixie maintained chart would also allow everyone to focus efforts on improving the official chart instead of their preferred third party one.

**Comment by l-rossetti** [comment]
any news on this? pixie cloud installation and its dependencies are not really automatable

**Comment by sjdrc** [comment]
Given that Pixie has not provided a helm chart or responded to any requests as to whether one will be supported, I think they they do not want to support a non-convoluted deployment of Pixie cloud to push people to using their managed service.

**Comment by vrabbi** [comment]
Any updates???

**Comment by tuwid** [comment]
@sjdrc thats true! I have created my own chart but its almost impossible to setup due to these dark patterns they have deployed in order for you to use their managed service, there's hardcoded values messing up the oauth and I've been bashing my head on the keyboard for the last 3 days due to this.

**Comment by lucascicco** [comment]
I went through the same process, @tuwid. Developing the chart was purely "reverse engineering", where I needed to analyze the source code to understand how each service should be implemented.

I believe the community should manage the Helm charts in a shared repository, instead of contributing directly to the Pixie repository, given that this PR has been open for over 3 years.

If you're willing to do that, let me know.
